sp800-63a/sec9_usability.md

```diff
@@ -6,34 +6,36 @@
 
 _This section is informative._
 
-[ISO/IEC 9241-11](#ISO9241-11) defines usability as: Extent to which a product can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use. This definition focuses on users, goals, and context of use as key elements necessary for achieving effectiveness, efficiency, and satisfaction. A holistic approach considering these key elements is necessary to achieve usability. 
+[ISO/IEC 9241-11](#ISO9241-11) defines usability as the "extent to which a product can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use." This definition focuses on users, goals, and context of use as key elements necessary for achieving effectiveness, efficiency, and satisfaction. A holistic approach considering these key elements is necessary to achieve usability. 
 
-The overarching goal of usability for enrollment and identity proofing is to promote a smooth, positive enrollment process for users by minimizing user burden (e.g. time and frustration) and enrollment friction (e.g. the number of steps to complete and amount of information to track). 
+The overarching goal of usability for enrollment and identity proofing is to promote a smooth, positive enrollment process for users by minimizing user burden (e.g., time and frustration) and enrollment friction (e.g., the number of steps to complete and amount of information to track). 
 
 The enrollment and identity proofing process sets the stage for a user’s interactions with a given authenticator for access to an online service; since negative first impressions can influence user perception of subsequent interactions, organizations benefit from promoting a positive user experience throughout the process. 
 
 An organization’s first step to promoting a smooth, positive enrollment process is familiarizing itself with the user. This consists of matching users with suitable CSPs based on their characteristics ahead of the enrollment and identity proofing process. 
 
+>**MG: this is kind of a federation comment. Here we're only really talking about a single CSP. Worth editing? **
```

Comment:
IMO this is OK -- -A addresses enrollment whether or not the CSP is going to be federating or acting directly as an RP.